dbflute-runtime/src/main/java/org/dbflute/cbean/coption/LikeSearchOption.java

```diff
@@ -55,6 +56,7 @@
     protected List<String> _originalWildCardList;
     protected List<SpecifiedColumn> _compoundColumnList;
     protected List<Integer> _compoundColumnSizeList;
+    protected boolean _treatCompoundColumnNullAsEmpty;
```

Comment:
change _treatCompoundColumnNullAsEmpty to _treatsCompoundColumnNullAsEmpty
("treat" to "treats")